dev/tasks/r/github.devdocs.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       {{ macros.github_checkout_arrow()|indent }}
 
       - uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: '4.1'
```

Comment:
Should we leave a comment here explaining why we're pinning a version? Link to a jira to change that?